eslint-rules/tests/component_prop_deprecation.json

```diff
@@ -78,16 +78,5 @@
         "fix": {"propName": "subtitle"}
       }
     ]
-  }, 
-  {
-    "component": "Picker",
-    "source": "module-with-deprecations",
-    "props": [
-      {
-        "prop": "migrate",
-        "message": "Please make sure to pass the 'migrate' prop.",
-        "isRequired": true
-      }
-    ]
```

Comment:
This should be taken as an example, it can stay here even after the removal of the `migrate` prop (or you can add a different example instead; this also fails a test that is run on this package)